client/www/pages/docs/strong-init.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Strong Init (experimental)
+---
+
+We're piloting strictly-typed variants of our client libraries that leverage your `instant.schema.ts` definition. We're calling it "Strong Init". Strong Init enhances InstaQL and InstaML methods with deep typesafety and enhances your editor experience with rich intellisence.
+
+If you want to migrate, simply swap out `init` with `init_experimental` and provide it your app's graph schema object.
```

Comment:
If you want to try it out, swap `init` with `init_experimental`, and pass in your 'schema' object. Here's how that looks: